exercises/acronym/example.go

```diff
@@ -0,0 +1,21 @@
+package acronym
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+const testVersion = 1
+
+func abbreviate(s string) (string, error) {
+	regex, _ := regexp.Compile("[A-Z]+[a-z]*|[a-z]+")
```

Comment:
I suggest [`MustCompile`](https://golang.org/pkg/regexp/#MustCompile) rather than `Compile` and ignore error to signal intent that you do expect this to compile.